config/crd/bases/crunchybridge.crunchydata.com_bridgeclusters.yaml

```diff
@@ -34,100 +34,103 @@ spec:
           metadata:
             type: object
           spec:
-            description: defines the desired state of BridgeCluster
+            description: BridgeClusterSpec defines the desired state of BridgeCluster
```

Comment:
I'm guessing there isn't a way to get the SDK to skip the variable identification through flags, or is there?
This is a clash between the auto-generation and GoDoc conventions. If there isn't a way to fix this with flags, I'd rather this file be reverted to head and I'll fix the comments so the auto generation produces the desired results.
In GoDoc conventions, the comment leads with the variable name, in the generated yaml, the json variable name may not be the same as the variable name, leading to confusion.

Reply:
reverted. I will and let you know if there is any flag to skip variable identification,  not my knowledge at the moment